cmd/scality-cosi-driver/cmd.go

```diff
@@ -20,21 +20,32 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"strings"
+	"time"
 
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/scality/cosi-driver/pkg/driver"
 	"k8s.io/klog/v2"
 
 	"github.com/scality/cosi-driver/pkg/grpcfactory"
+	"github.com/scality/cosi-driver/pkg/metrics"
 )
 
 const (
-	provisionerName     = "scality.com"
-	defaultDriverPrefix = "cosi"
+	provisionerName       = "scality.com"
+	defaultDriverAddress  = "unix:///var/lib/cosi/cosi.sock"
+	defaultDriverPrefix   = "cosi"
+	defaultMetricsPath    = "/metrics"
+	defaultMetricsPrefix  = "scality_cosi_driver"
```

Comment:
```suggestion
	defaultMetricsPrefix  = "scality_cosi_driver_"
```

Reply:
Maybe we should check if value is overrided with an empty string, metrics should not be like `_s3_request_total`, or maybe empty string should not be allowed